main.rb

```diff
@@ -4,37 +4,172 @@
 require './task.rb'
 require 'date'
 require 'bcrypt'
+require './login.rb'
+require './sign_up.rb'
+require './task_manager.rb'
 
 connection = DatabaseConnection.connection
 table_setup = TableSetup.new(connection)
 table_setup.setup_database
 
-def for_admin(email)
+def assign_task(admin)
```

Comment:
this method should be inside task_manager

Reply:
def assign_task(user)
  ## show error if user is not admin
end

in user class
def admin?
 admin
end